### PR TITLE
Use InDelta in TestExternallyControlledRun 

### PR DESCRIPTION
**Pull request by MStoykov**

### CODE REVIEW

**Review by na-- — APPROVED** [review]

**Review by imiric — APPROVED** [review]
We shouldn't have to do this, but OK 🙁

File: lib/executor/constant_arrival_rate_test.go [hunk 1]
```diff
@@ -227,7 +228,7 @@ func TestConstantArrivalRateRunCorrectTiming(t *testing.T) {
 				// check that we got around the amount of VU iterations as we would expect
 				var currentCount int64
 
-				for i := 0; i < 3; i++ {
+				for i := 0; i < seconds; i++ {
```
Comment (na--) [inline lib/executor/constant_arrival_rate_test.go:231]: Erm, I'm confused - won't `seconds` be `0`, so this loop will never execute? :confused:

**Review by na-- — APPROVED** [review]
LGTM

**Comment by na--** [comment]
I added this to the v0.29.0 since it fixes the `context cancelled` bug (https://github.com/loadimpact/k6/issues/1485)

**Pull request merged**